packages/expo-ui/ios/ListView.swift

```diff
@@ -0,0 +1,18 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import ExpoModulesCore
+import SwiftUI
+
+class ListViewProps: ExpoSwiftUI.ViewProps {}
+
+struct ListView: ExpoSwiftUI.View {
+  @EnvironmentObject var props: ListViewProps
+
+  var body: some View {
+    List {
+      UnwrappedChildren { child, _ in
```

Comment:
You should be able to just use UnwrappedChildren() with no params here

Reply:
Thank you!